plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -297,9 +297,10 @@ private void createAndPopulateTooManyRowsTable(TestingKafka kafka, TestingPinotC
                     .set("updatedAt", initialUpdatedAt.plusMillis(i * 1000).toEpochMilli())
                     .build()));
         }
+        // Update for pinot 0.11.0: rows with null time column values are ingested
         // Add a null row, verify it was not ingested as pinot does not accept null time column values.
         // The data is verified in testBrokerQueryWithTooManyRowsForSegmentQuery
-        tooManyRowsRecordsBuilder.add(new ProducerRecord<>(TOO_MANY_ROWS_TABLE, "key" + MAX_ROWS_PER_SPLIT_FOR_SEGMENT_QUERIES, new GenericRecordBuilder(tooManyRowsAvroSchema).build()));
+        //tooManyRowsRecordsBuilder.add(new ProducerRecord<>(TOO_MANY_ROWS_TABLE, "key" + MAX_ROWS_PER_SPLIT_FOR_SEGMENT_QUERIES, new GenericRecordBuilder(tooManyRowsAvroSchema).build()));
```

Comment:
Probably extract this specific case to new test method and override it in <0.11?